### DISCUSSION TITLE
TeslaFi

### DISCUSSION THREAD

**Discussion by V2Kot** [marcone/teslausb#815] [Q&A]
I've noticed that the config file on GitHub has a reference to the TeslaFi (no reference in installation config) Is it relevant and can be used, to integrate with TeslaUSB to keep the care awake?

**Comment by THX723** [comment]
Yes, it's implemented. Just follow the direction in the sample config file.

**Reply by V2Kot** [reply]
Tested today, works perfectly keeping the car awake during archiving.
Thanks, solves the issue with power drainage.
